site/pages/[locale]/token-revokes.js

```diff
@@ -372,61 +399,20 @@ const TokenRevokes = function () {
         text={t('utilities-text.token-revokes')}
         title={t('list-and-revoke-token-approvals')}
       >
-        {/* {tokenApprovals.length > 0 && (
-          <section className="flex flex-col">
-            <div className="grid grid-cols-4">
-              <span className="m-auto text-gray-600 font-bold">
-                {t('token')}
-              </span>
-              <span className="hidden m-auto text-gray-600 font-bold md:block">
-                {t('spender-address')}
-              </span>
-              <span className="m-auto text-gray-600 font-bold">
-                {t('allowance')} / {t('balance')}
-              </span>
-              <span className="m-auto text-gray-600 font-bold">
-                {t('actions')}
-              </span>
-              {tokenApprovals.map(
-                ({ address, allowance, transactionHash, spender }) => (
-                  <React.Fragment key={transactionHash}>
-                    <Token address={address} />
-                    <div className="hidden my-auto md:block">
-                      <Token address={spender} />
-                    </div>
-                    <div className="my-auto">
-                      <Allowance address={address} data={allowance} />
-                      <span> / </span>
-                      <Balance address={address} />
-                    </div>
-                    <Button
-                      disabled={!active}
-                      onClick={() => handleRevoke(address, spender)}
-                      width="w-28"
-                    >
-                      {t('revoke')}
-                    </Button>
-                  </React.Fragment>
-                )
-              )}
-            </div>
-          </section>
-        )} */}
-
-        {tokenApprovals.length > 0 && (
-          <div className="w-100 md:w-150 overflow-scroll lg:w-full">
-            <table className="w-150">
-              <thead className="">
-                <tr className="bg-slate-50 text-slate-600 border-slate-200 rounded-xl border text-left text-sm font-light">
-                  <th className="w-10 px-2 py-4 font-medium">{t('token')}</th>
-                  <th className="w-14 font-medium">{t('allowance')}</th>
-                  <th className="w-14 font-medium">{t('balance')}</th>
-                  <th className="w-24 font-medium">{t('spender-address')}</th>
-                  <th className="w-10 font-medium">{t('actions')}</th>
-                </tr>
-              </thead>
-              <tbody>
-                {tokenApprovals.map(
+        <div className="w-100 md:w-150 overflow-scroll lg:w-full">
```

Comment:
I think these overflow should be as `auto` so the scroll is invisible if not needed (in the screenshots, they are shown even though the table is empty, and therefore, not needed)

```suggestion
        <div className="w-100 md:w-150 overflow-auto lg:w-full">
```

Reply:
fixed